public/paysagerenderer/main.js

```diff
@@ -42,11 +43,36 @@ var Paysage = window.Paysage || {};
     });
 
     installResizeHandler();
+
+    var urlHash = '';
+    setInterval(function () {
```

Comment:
It would be better to register for the hash change event instead of polling
https://developer.mozilla.org/fr/docs/Web/API/WindowEventHandlers/onhashchange